sdk/core/src/http_client.rs

```diff
@@ -139,6 +208,7 @@ fn get_version(_response: &reqwest::Response) -> Option<http::Version> {
 
 #[cfg(feature = "enable_reqwest")]
 #[cfg(not(target_arch = "wasm32"))]
+#[allow(dead_code)]
```

Comment:
Why not delete this?

Reply:
It was a workaround introduced by Cameron because reqwest in wasm does not return the version. We do not use it anyway so I've just deleted it.